src/pages/alternatives.js

```diff
@@ -2,16 +2,53 @@ import React from 'react';
 import Link from 'gatsby-link';
 import Img from 'gatsby-image';
 
+import PrimaryButton from '../components/PrimaryButton';
+
+import styles from './alternatives.module.scss';
+import sharedStyles from '../assets/shared-styles/shared.module.scss';
+import classNames from 'classnames';
+
 const AlternativesPage = ({ data }) => (
   <div>
     <Img
       alt="Text overlay that reads: You are not alone. There is Help available. Background image of man sitting on a wall looking sad and thoughtful."
       sizes={data.alternativesImage.sizes}
     />
-    <h1>Alternatives</h1>
-    <p>{ data.site.siteMetadata.contact_website }</p>
-    <p>{ data.site.siteMetadata.contact_url }</p>
-    <p>{ data.site.siteMetadata.contact_phone }</p>
+    <div className={sharedStyles.pageWrapper}>
+      <div>
+        <h1 className={styles.statPercentage}>49%</h1>
+        <p className={styles.statCaption}>of sex buyers have never told anyone they bought sex</p>
+      </div>
+      <div className={classNames(styles.quote, styles.blueBackground)}>
+        <p className={styles.statement}>"It's such a relief to be out of that life cycle."</p>
+        <p className={styles.signature}>-former sex buyer</p>
+      </div>
+      <div>
+        <h1 className={styles.statPercentage}>64%</h1>
+        <p className={styles.statCaption}>of sex buyers in the United States said they want to stop</p>
+      </div>
+      <div className={classNames(styles.quote, styles.tealBackground)}>
+        <p className={styles.statement}>"I learned that the harms of prostitution go much deeper than what appears on the surface."</p>
+        <p className={styles.signature}>-fomer sex buyer</p>
+      </div>
+      <div>
+        <p className={styles.choice}>Make the choice without regret...</p>
+        <p className={styles.contactUs}>contact us today.</p>
+      </div>
+      <div>
+        <PrimaryButton url={ data.site.siteMetadata.contact_url }>Learn More</PrimaryButton>
+      </div>
+      <div>
+        <p className={styles.information}>For more information, call</p>
+        <p className={styles.link}>{ data.site.siteMetadata.contact_phone }</p>
```

Comment:
How about making this phone number a click-to-call link? [Like this](https://css-tricks.com/the-current-state-of-telephone-links/)

Reply:
Done! Also made a small change and removed the blue underline from links!

Reply:
Awesome!